packages/side-nav/src/SideNav.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ulStyleOverrides } from './styles';
+
+interface SideNavProps {
+  /**
+   * Class name that will be applied to SideNav root component.
+   */
+  className?: string;
+  /**
+   * Content that will appear inside of SideNav root component.
+   */
+  children?: React.ReactNode;
+}
+
+/**
+ * # SideNav
+ *
+ * ```
+<SideNav>
+  <SideNavGroup headerText="Section Header">
+    <SideNavItem>
+      Link Content
+    </SideNavItem>
+  </SideNavGroup>
+</SideNav>
+ * ```
+ * @param props.className Class name that will be applied to SideNav root component.
+ * @param props.children Content that will appear inside of SideNav root component.
+ *
+ */
+function SideNav({ children, ...rest }: SideNavProps) {
+  return (
+    <nav style={{ width: '200px' }} aria-label="side-nav" {...rest}>
```

Comment:
can we move this to a className with our emotion instance

Reply:
Haha, @DesignerDave asked the opposite and said to inline it, so I got rid of that. I actually like to have it as a class instead and it's mostly a nitpick, so I'm going to revert it back. Sorry Dave!

Reply:
I think maybe he meant `<nav className={css`width: 200px`} />` but perhaps I'm wrong. Either way definitely support using className rather than style 👍